presto-delta/pom.xml

```diff
@@ -0,0 +1,290 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.facebook.presto</groupId>
+        <artifactId>presto-root</artifactId>
+        <version>0.264-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>presto-delta</artifactId>
+    <description>Presto - Delta Table Connector</description>
+    <packaging>presto-plugin</packaging>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+        <enforcer.skip>true</enforcer.skip>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.facebook.presto.delta</groupId>
+            <artifactId>presto-delta-driver</artifactId>
+            <classifier>shaded</classifier>
+            <version>${project.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>io.delta</groupId>
+                    <artifactId>delta-standalone_2.12</artifactId>
+                </exclusion>
```

Comment:
Exclusion of this will cause IDE showing cannot resolve symbol error. It might be better to set the scope of delta-standalone dependency as compile in driver module, and then remove this execlusion.

Reply:
@artiship If you do `mvn clean install -DskipTests` before importing the project into Intellij they should be resolvable in Intellij.

Reply:
@vkorukanti Since delta tables are also been managed by hive metastore. Is it possible for the user to write SQL joining delta tables and hive tables in the same catalog?

Reply:
@artiship It is not possible in the same Delta catalog, but you can add another catalog (type Hive) which talks to the same metastore and do join.

Reply:
> If you do mvn clean install -DskipTests before importing the project into Intellij they should be resolvable in Intellij.

@vkorukanti Due to the exclusion of this dependency, IntelliJ isn't able to resolve the imports from Delta Standalone Reader library, so I tried your suggestion but IntelliJ is still not able to resolve. We may have to think of an alternative here.

Reply:
Hey @imjalpreet, don't worry about this. The shaded module dependency will go away once Delta 0.3 library version is released (sometime in the next month). With 0.3 version, `presto-delta` library can depend directly on the Delta library.

Reply:
Pushed the change for Delta client library version upgrade. It also removes the shading module. Now the IDE and test should work properly in CI. @artiship @imjalpreet @nmahadevuni